docs/HowToSupportProtectedBuild.md

```diff
@@ -38,14 +40,48 @@
 ## Steps for Supporting Protected Build in a new chipset
 The following sections give detailed explanation regarding some of the steps / code changes involved in supporting protected build on a new Chipset.
 
-### Make.defs
+### Prerequisites
+- TizenRT flat build needs to be successfully ported and tested on the target CHIPSET board.
+- The developer should have information about the types of memories supported on the board, along with their start addresses and sizes. The partition information of the flash memory should also be known. These details can be found in either the chipset's data sheet, project configuration file, linker or download scripts or in the memory map mentioned as part of the chip's source code.
+
+### Creating a new configuration for Protected Build
+- Every chipset has a hello configuration under path build/config/CHIPSET_NAME.
+- Copy the contents of the "hello" folder into a new folder called “hello_protected”
+- Modify the defconfig and Make.defs as follows:
+
+#### Defconfig
+- The following configs need to be set or unset in the defconfig file based on ARMV7M or ARMV8M architecture supported.
+
+```
+CONFIG_BUILD_FLAT is not set
+CONFIG_BUILD_PROTECTED=y
+CONFIG_BUILD_2PASS=y
+CONFIG_PASS1_TARGET="all"
+CONFIG_PASS1_OBJECT=""
+CONFIG_ARM_HAVE_MPU_UNIFIED=y
+CONFIG_ARMV7M_MPU=y
+CONFIG_ARMV7M_MPU_NREGIONS=8
+CONFIG_MM_KERNEL_HEAP=y
+CONFIG_LIB_SYSCALL=y
+CONFIG_SYS_RESERVED=8
+CONFIG_SYS_NNEST=2
```

Comment:
1. Instead of them, could you provide the path in menuconfig to enable them?
2. All of config should be controlled using menuconfig. But  before done below, it is impossible. Let's move `### creating a new configuration for protected build` at end of steps.
3. Please provide the way how to make a defconfig.
After setting configs using menuconfig, copy os/.config file to configs folder with defconfig named.

Reply:
1.  Yes updated
2. I have moved the new configuration part down.
3. Added.